src/main/webapp/resources/js/pages/analysis/components/tree/Tree.jsx

```diff
@@ -81,7 +81,7 @@ export default function Tree() {
         ) : (
           <div>
             <ButtonGroupWrapper>
-              <Radio.Group value={currTreeShape} onChange={e => handleClick(e)}>
+              <Radio.Group value={currTreeShape} onChange={e => handleClick(e)} id="t-tree-shape-tools">
```

Comment:
`onChange={e => handleClick(e)}` can be written `{handleClick}`

Reply:
Updated in b7f397facd7f75e5f8d51ba00461f3f813291d2f